Description: Fix several spelling errors
 This last bit of the bug report seems to have been missed.
 Also some new spelling errors and another pod error
Author: Florian Ernst <florian@debian.org>
Bug: https://rt.cpan.org/Ticket/Display.html?id=60282
Last-Update: 2011-07-16
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/lib/HTML/Template.pm
+++ b/lib/HTML/Template.pm
@@ -298,7 +298,7 @@
 directly.  See below for more information on C<HTML_TEMPLATE_ROOT>
 and the C<path> option to C<new()>.
 
-As a protection against infinitly recursive includes, an arbitrary
+As a protection against infinitely recursive includes, an arbitrary
 limit of 10 levels deep is imposed.  You can alter this limit with the
 C<max_includes> option.  See the entry for the C<max_includes> option
 below for more details.
@@ -2673,6 +2673,8 @@
 An error occurs if you try to set a value that is tainted if the C<force_untaint>
 option is set.
 
+=back
+
 =cut
 
 sub param {
--- a/lib/HTML/Template/FAQ.pm
+++ b/lib/HTML/Template/FAQ.pm
@@ -113,7 +113,7 @@
 
 Short answer: you can't. Longer answer: you shouldn't since this violates
 the fundamental concept behind L<HTML::Template> - that design and code
-should be seperate.
+should be separate.
 
 But, inevitably some people still want to do it. If that describes
 you then you should take a look at L<HTML::Template::Expr>. Using
